pysph/base/stratified_sfc_gpu_nnps.pyx

```diff
@@ -102,15 +105,16 @@ cdef class StratifiedSFCGPUNNPS(GPUNNPS):
     cpdef _bin(self, int pa_index):
         cdef NNPSParticleArrayWrapper pa_wrapper = self.pa_wrappers[pa_index]
 
-        fill_pids = self.helper.get_kernel("fill_pids")
+        fill_pids = profile_kernel(*self.helper.get_kernel("fill_pids"))
```

Comment:
Do you want the helper to always return a profiled kernel?

Reply:
These should not call `profile_kernel` as the helper only returns a profiled kernel, right?